chapters/statements.tex

```diff
@@ -46,7 +46,7 @@ \subsection{Execution of an algorithm in a model}\label{execution-of-an-algorith
   is initialized with its start-value (i.e.\ the value of the \lstinline!start! attribute)
   provided the start-value can be evaluated before evaluating the initial algorithm
   (a literal value can always be evaluated); otherwise a default start-value is used
-  as if the \lstinline!start! attribute was not set.
+  (together with appropriate diagnostics) as if the \lstinline!start! attribute was not set.
```

Comment:
I don't find this sufficient.  If one wants the _appropriate diagnostics_, I think one should rely on a non-stringent tool that to deliver non-standard behavior (together with appropriate diagnostics).  From the specification's point of view, on the other hand, I'd say treating the inability to apply provided `start` attributes as an error is the only reasonable alternative.